cruise/CruiseInformation.csv

```diff
@@ -1,54 +1,75 @@
-KN-217,R/V Knorr,2014-04-11T17:00:00,2014-04-19T10:30:00,Pioneer 2
-KN-218,R/V Knorr,2014-05-01T00:00:00,2014-05-12T23:59:59,Pioneer glider cp_387 recovery
-MV-1404,R/V Melville,2014-06-10T00:00:00,2014-06-29T23:59:59,Papa 2
+KN-217,R/V Knorr,2014-04-11T17:00:00,2014-04-19T10:30:00,Pioneer 2 #TO BE DELETED
+KN217,R/V Knorr,2014-04-11T17:00:00,2014-04-19T10:30:00,Pioneer 2
+KN-218,R/V Knorr,2014-05-01T00:00:00,2014-05-12T23:59:59,Pioneer glider cp_387 recovery #TO BE DELETED
+KN218,R/V Knorr,2014-05-01T00:00:00,2014-05-12T23:59:59,Pioneer glider cp_387 recovery
+MV-1404,R/V Melville,2014-06-10T00:00:00,2014-06-29T23:59:59,Papa 2 #TO BE DELETED
+MV1404,R/V Melville,2014-06-10T00:00:00,2014-06-29T23:59:59,Papa 2
 CT20140711,R/V Connecticut,2014-07-11T00:00:00,2014-07-11T23:59:59,Pioneer 2a 
-TN-313,R/V Thomas G. Thompson,2014-07-13T00:00:00,2014-10-06T23:59:59,Cabled (rvdata)
+TN-313,R/V Thomas G. Thompson,2014-07-13T00:00:00,2014-10-06T23:59:59,Cabled (rvdata) #TO BE DELETED
+TN313,R/V Thomas G. Thompson,2014-07-13T00:00:00,2014-10-06T23:59:59,Cabled (rvdata)
 EK20140807,R/V Elakha,2014-08-07T00:00:00,2014-08-07T00:00:00,Endurance Glider Deployment
 TY-1408,CCGS Tully,2014-08-29T00:00:00,2014-08-29T00:00:00,Papa Glider Operations
-KN-221-4,R/V Knorr,2014-09-07T09:00:00,2014-09-28T23:59:59,Irminger 1
-KN-222,R/V Knorr,2014-10-03T22:00:00,2014-10-11T11:00:00,Pioneer 3 Leg 1
+KN-221-4,R/V Knorr,2014-09-07T09:00:00,2014-09-28T23:59:59,Irminger 1 #TO BE DELETED
+KN221-04,R/V Knorr,2014-09-07T09:00:00,2014-09-28T23:59:59,Irminger 1
+KN-222,R/V Knorr,2014-10-03T22:00:00,2014-10-11T11:00:00,Pioneer 3 Leg 1 #TO BE DELETED
+KN222,R/V Knorr,2014-10-03T22:00:00,2014-10-11T11:00:00,Pioneer 3 Leg 1
 OC1410A,R/V Oceanus,2014-10-06T00:00:00,2014-10-10T23:59:59,Endurance 2 (rvdata)
 AQ20141030,R/V Aquila,2014-10-30T00:00:00,2014-10-31T23:59:59,Papa Glider gp_453 recovery
-KN-224,R/V Knorr,2014-12-11T19:00:00,2014-12-16T23:59:59,Pioneer 3 leg 2
+KN-224,R/V Knorr,2014-12-11T19:00:00,2014-12-16T23:59:59,Pioneer 3 leg 2 #TO BE DELETED
+KN224,R/V Knorr,2014-12-11T19:00:00,2014-12-16T23:59:59,Pioneer 3 leg 2
 WJ20141230,M/V Warren Jr.,2014-12-30T13:30:00,2014-12-31T12:00:00,Pioneer ISSM-1 buoy recovery
 SI-1501,M/V Scarlett Isabella,2015-01-10T00:00:00,2015-01-13T23:59:59,#TO BE DELETED
 SI20150110,M/V Scarlett Isabella,2015-01-10T00:00:00,2015-01-13T23:59:59,Pioneer 3a
 PS1501,R/V Pacific Storm,2015-01-21T20:28:00,2015-01-21T22:02:00,Endurance GL info
-AT-26-29,R/V Atlantis,2015-02-12T15:00:00,2015-03-04T23:59:59,Southern Ocean 1
+AT-26-29,R/V Atlantis,2015-02-12T15:00:00,2015-03-04T23:59:59,Southern Ocean 1 #TO BE DELETED
+AT26-29,R/V Atlantis,2015-02-12T15:00:00,2015-03-04T23:59:59,Southern Ocean 1
 EN552,R/V Endeavor,2015-03-01T00:00:00,2015-03-05T23:59:59,Pioneer glider cp_387 recovery
 MM20150327,R/V Maria S. Merian,2015-03-27T00:00:00,2015-03-27T23:59:59,Pioneer glider cp_336 recovery

```

Comment:
This should be after EK-1503 to be chronological.

Reply:
done

Reply:
looks good